### PR TITLE
More channel edits - names - logos - group-titles

### PR DESCRIPTION
**Pull request by jkemp814**
The list still shows there are channels that need edits to fill in missing logos and consistency across channels with multiple streams.

### CODE REVIEW

File: channels/us.m3u [hunk 1]
```diff
@@ -1272,13 +1272,13 @@ http://livestreamcdn.net:1935/ExtremaTV/ExtremaTV/playlist.m3u8
 https://cdn20.liveonlineservices.com/hls/eye95tv.m3u8
 #EXTINF:-1 tvg-id="FailArmy.us" tvg-name="FailArmy" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/nbzbHum.png" group-title="Comedy",FailArmy
 https://9c17e762ec814557b3516dd3e0a7ca56.mediatailor.us-east-1.amazonaws.com/v1/manifest/44f73ba4d03e9607dcd9bebdcb8494d86964f1d8/Plex-intl_FailArmy/ade59f27-be9b-4b71-b8dc-05928c1dfdf4/2.m3u8
-#EXTINF:-1 tvg-id="FailArmy.us" tvg-name="FailArmy" tvg-country="US" tvg-language="English" tvg-logo="https://assets.tvplayer.com/tvos/images/channels/585.png" group-title="General",FailArmy (720p)
+#EXTINF:-1 tvg-id="FailArmy.us" tvg-name="FailArmy" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/nbzbHum.png" group-title="General",FailArmy (720p)
```
Comment (Dum4G) [inline channels/us.m3u:1275]: Comedy

**Review by Dum4G — CHANGES_REQUESTED** [review]

**Comment by Dum4G** [comment]
Please check the US playlist for plex wurl links. AMC Weddings from them is not working for me.

File: channels/us.m3u (hunk 1 above)
Comment (jkemp814) [reply channels/us.m3u:1275]: I was in the process once I had seen it on GitHub.

**Comment by jkemp814** [comment]
I don't have any AMC All Weddings in VLC or Kodi

**Comment by jkemp814** [comment]
I did find an All Weddings with the plex.wurl in the link

[hls @ 0x7f6050000b80] Can't support the subtitle(uri: subtitles.m3u8)
[https @ 0x7f6050343ac0] HTTP error 403 Forbidden

File: channels/us.m3u [hunk 2]
```diff
@@ -1536,11 +1536,11 @@ https://fueltv-fueltv-13-se.samsung.wurl.com/manifest/playlist.m3u8
 http://104.143.4.5:2080/funroads.m3u8
 #EXTINF:-1 tvg-id="FunZoneBumblebee.us" tvg-name="Fun Zone (Bumblebee)" tvg-country="US" tvg-language="English" tvg-logo="https://imgix.unreel.me/https%3A%2F%2Fmedia.unreel.me%2Fprod%2Fbumblebeetv%2Flogo%2F319472bf-d953-4dbd-a197-b7aed15b7fa9?w=80&amp;h=45&amp;fit=fit&amp;blur=0&amp;ixlib=js-1.2.0&amp;s=6f4f09e01cf6423d194abdb11e81a61f" group-title="",Fun Zone (Bumblebee)
 https://stitcheraws.unreel.me/wse-node02.powr.com/live/5e2625030145130f25474622/playlist.m3u8
-#EXTINF:-1 tvg-id="FunnyOrDie.us" tvg-name="Funny Or Die" tvg-country="US" tvg-language="English" tvg-logo="https://od.lk/s/MF8yMzE5OTIyMjBf/Redbox_512x512.png" group-title="",Funny Or Die
+#EXTINF:-1 tvg-id="FunnyorDie.us" tvg-name="Funny or Die Die" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/PMMsywm.png" group-title="Comedy",Funny or Die
```
Comment (Dum4G) [inline channels/us.m3u:1539]: Die die

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/us.m3u [hunk 3]
```diff
@@ -1606,9 +1606,9 @@ https://glewedtv-3.plex.wurl.com/manifest/playlist.m3u8
...
 https://vcngfcssai.teleosmedia.com/linear/globalfashionchannel/globalfashionchannel/playlist.m3u8
-#EXTINF:-1 tvg-id="GlobalGotTalent.us" tvg-name="Global Got Talent" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/QHiOf7Z.png" group-title="",Global Got Talent
+#EXTINF:-1 tvg-id="GlobalGotTalent.us" tvg-name="Global Got Talent" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/QHiOf7Z.png" group-title="Entertainment",Global Got Talent
 https://dai2.xumo.com/amagi_hls_data_xumo1212A-rokugottalentglobal/CDN/playlist.m3u8
-#EXTINF:-1 tvg-id="GlobalGotTalent.us" tvg-name="Global Got Talent" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/TheRokuChannel_880x880.png?raw=true" group-title="",Global Got Talent
+#EXTINF:-1 tvg-id="GlobalGotTalent.us" tvg-name="Global Got Talent" tvg-country="US" tvg-language="English" tvg-logo=https://i.imgur.com/QHiOf7Z.png" group-title="Entertainment",Global Got Talent
```
Comment (Dum4G) [inline channels/us.m3u:1611]: " is missing

**Review by Dum4G — CHANGES_REQUESTED** [review]

**Review by BellezaEmporium — APPROVED** [review]
Sounds good to me !

**Review by Dum4G — APPROVED** [review]

**Pull request merged**